pkg/handlers/v1/subnet.go

```diff
@@ -0,0 +1,77 @@
+package v1
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+)
+
+type subnetConfiguration struct {
+	CIDRBlock string `json:"cidrBlock"`
+}
+
+type subnetConfigurationDiff struct {
+	PreviousValue *subnetConfiguration `json:"previousValue"`
+	UpdatedValue  *subnetConfiguration `json:"updatedValue"`
+	ChangeType    string               `json:"changeType"`
+}
+
+type subnetTransformer struct{}
+
+func (t subnetTransformer) Create(event awsConfigEvent) (Output, bool, error) {
```

Comment:
is **awsConfigEvent** our custom object type or this is something we get from AWS Config event? Again, moving from Java to Go naming convention is throwing me off. Is there a good reference on naming conventions for Go? I see certain structs like **Output** being a title case but others are camelcase.

Reply:
awsConfigEvent is defined in [config.go](https://github.com/asecurityteam/awsconfig-transformerd/blob/e76dfa1da2cff43bd8d3b1e978f54f0bff04304a/pkg/handlers/v1/config.go#L40).

Reply:
Here **awsConfigEvent** does not seem like a local struct. Shouldn't this be title case then? Sorry, there are nitpick comments trying to understand Go conventions. Please dont block your change on my comments.